.github/workflows/update-docker-args.yml

```
@@ -0,0 +1,58 @@
+name: Update Docker Args
+on:
+  schedule:
+    - cron: 0 1 * * 1
+  workflow_dispatch:
+jobs:
+  update-docker-args:
+    runs-on: ubuntu-22.04
+    permissions:
+      contents: write
+      pull-requests: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Update Docker Args
+        shell: bash
+        id: args
+        run: |
+          BASE_IMAGE="gcr.io/distroless/cc-debian12"
+          BASE_IMAGE_TAG="debug"
+          BASE_IMAGE_DIGEST=$(gcloud container images list-tags $BASE_IMAGE --format json | jq --arg tag "$BASE_IMAGE_TAG" -r '.[] | {digest: .digest, tag: .tags[]} | select(.tag == $tag) | .digest')
+          if [ -n "$BASE_IMAGE_DIGEST" ]; then
+            echo -n "${IMAGE}:${TAG}@${BASE_IMAGE_DIGEST}" > java-11/base.image
+            echo -n "${IMAGE}:${TAG}@${BASE_IMAGE_DIGEST}" > java-21/base.image
```

If I'm following correctly, typo:
```suggestion
            echo -n "${BASE_IMAGE}:${BASE_IMAGE_TAG}@${BASE_IMAGE_DIGEST}" > java-11/base.image
            echo -n "${BASE_IMAGE}:${BASE_IMAGE_TAG}@${BASE_IMAGE_DIGEST}" > java-21/base.image
```